crates/turbopack-ecmascript-runtime/js/src/build/runtime.ts

```diff
@@ -95,10 +95,7 @@ function loadChunkPath(chunkPath: ChunkPath): void {
   }
 }
 
-async function loadChunkAsync(
-  source: SourceInfo,
-  chunkData: ChunkData
-): Promise<any> {
+async function loadChunkAsync(chunkData: ChunkData): Promise<any> {
```

Comment:
Instead of removing this argument, I think it might be better to actually use it and add this information to the error.

Reply:
done